module/apmhttp/client.go

```diff
@@ -104,7 +104,15 @@ func (r *roundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
 	}
 
 	name := r.requestName(req)
-	span := tx.StartSpan(name, r.spanType, apm.SpanFromContext(ctx))
+	var span *apm.Span
+	if r.traceRequests {
+		// Trace requests are enabled, use this as the parent span and mark the trace requests as
+		// exit spans.
+		span = tx.StartSpan(name, r.spanType, apm.SpanFromContext(ctx))
+	} else {
+		// No trace requests, mark the span as exit span.
+		span = tx.StartExitSpan(name, r.spanType, apm.SpanFromContext(ctx))
+	}
```

Comment:
See my other comment in clienttrace.go - I'm pretty sure this span should always be an exit span. Because the net/http/httptrace spans have the same type and subtype, only a different action, they can be children of the exit span.

Reply:
Good point! Updated!